scripts/jtsync/README.md

```diff
@@ -0,0 +1,58 @@
+# jtsync - synchronize status from jenkins to trello
+    Usage: jtsync --ci SERVICE (--matrix|--job) JOB_STATUS
+            --ci SERVICE                 Which ci is used (suse or opensuse)
+            --matrix NAME,PROJECT        Set status of a matrix job
+            --job NAME                   Set status of a normal job
+
+Matrix job cards are selected in this form:
+
+    $ jtsync --ci suse --matrix crowbar-trackupstream,Devel:Cloud:7:Staging
```

Comment:
@dirkmueller you absolutly right. Mhm. your right the name is cloud-trackupstream, I will change that to match the real names.

Yes I'm currently working on making the matrix jobs handle the case that one matrix job fails and the others are green.